CHANGELOG.md

```diff
@@ -2975,7 +2975,7 @@ https://iojs.org/api/tls.html
 - Added async session storage events.
 - Added async SNI callback.
 - Added multi-key server support (for example, ECDSA+RSA server).
-- Added optional callback to `checkServerIdentity` for manual certificate validation in user-land.
+- Added optional callback to `checkServerIdentity` for manual certificate validation in userland.
```

Comment:
Is it cool to change the CHANGELOG after the fact like this for what amounts to (at worst) a typo? I personally think it's totally fine but uh...anyone have a problem with it?